src/pageTypes/components/PageTypeList/PageTypeList.tsx

```diff
@@ -0,0 +1,145 @@
+    colName: {
+      paddingLeft: 0
+    },
+    link: {
+      cursor: "pointer"
+    }
+  }),
+  { name: "PageTypeList" }
+);
+
+interface PageTypeListProps
+  extends ListProps,
+    ListActions,
+    SortPage<PageTypeListUrlSortField> {
+  pageTypes: PageTypeList_pageTypes_edges_node[];
+}
+
+const numberOfColumns = 2;
+
+const PageTypeList: React.FC<PageTypeListProps> = props => {
+  const {
+    disabled,
+    pageTypes,
+    pageInfo,
+    onNextPage,
+    onPreviousPage,
+    onRowClick,
+    onSort,
+    isChecked,
+    selected,
+    sort,
+    toggle,
+    toggleAll,
+    toolbar
+  } = props;
+  const classes = useStyles(props);
+
+  return (
+    <ResponsiveTable>
+      <TableHead
+        colSpan={numberOfColumns}
+        selected={selected}
+        disabled={disabled}
+        items={pageTypes}
+        toggleAll={toggleAll}
+        toolbar={toolbar}
+      >
+        <TableCellHeader
+          direction={
+            sort.sort === PageTypeListUrlSortField.name
+              ? getArrowDirection(sort.asc)
+              : undefined
+          }
+          arrowPosition="right"
+          onClick={() => onSort(PageTypeListUrlSortField.name)}
+          className={classes.colName}
+        >
+          <FormattedMessage
+            defaultMessage="Content Type Name"
+            description="page type name"
+          />
+        </TableCellHeader>
+      </TableHead>
+      <TableFooter>
+        <TableRow>
+          <TablePagination
+            colSpan={numberOfColumns}
+            hasNextPage={pageInfo && !disabled ? pageInfo.hasNextPage : false}
+            onNextPage={onNextPage}
+            hasPreviousPage={
+              pageInfo && !disabled ? pageInfo.hasPreviousPage : false
+            }
+            onPreviousPage={onPreviousPage}
+          />
+        </TableRow>
+      </TableFooter>
+      <TableBody>
+        {renderCollection(
+          pageTypes,
+          pageType => {
+            const isSelected = pageType ? isChecked(pageType.id) : false;
+            return (
+              <TableRow
+                className={!!pageType ? classes.link : undefined}
+                hover={!!pageType}
+                key={pageType ? pageType.id : "skeleton"}
+                onClick={pageType ? onRowClick(pageType.id) : undefined}
+                selected={isSelected}
+                data-test="id"
+                data-test-id={pageType?.id}
+              >
+                <TableCell padding="checkbox">
+                  <Checkbox
+                    checked={isSelected}
+                    disabled={disabled}
+                    disableClickPropagation
+                    onChange={() => toggle(pageType.id)}
+                  />
+                </TableCell>
+                <TableCell className={classes.colName}>
+                  {pageType ? (
+                    <span data-test="name">{pageType.name}</span>
+                  ) : (
+                    <Skeleton />
+                  )}
```

Comment:
I'm not sure if it works as intended - `name` is a special field because it's in every fragment, but other properties can be `undefined` if `pageType` was partially fetched. If so, skeleton wouldn't be displayed.
It's okay here, but for other fields I'd recommend you to use `pageType?.name || <Skeleton />`